### ISSUE TITLE
object detection API: is the coco dataset performance measured on the commonly used minival set?

### ISSUE THREAD

**Issue by twangnh** [tensorflow/models#4421]
As stated in TF object detection model zoo:

> detector performance on subset of the COCO validation set or Open Images test split as measured by the dataset-specific mAP measure.

but do is this subset of COCO validation set the commonly used 5K minival set?

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by MLman** [comment]
I have the same question. 
How can I reproduce the object detection performance?
Which dataset exactly was used? 
The numbers are not reproducible.

 Have I written custom code: NO
OS Platform and Distribution: Ubuntu 16
TensorFlow installed from: Conda
TensorFlow version: 1.8 in python 3.6
Bazel version: N/A
CUDA/cuDNN version: 9.0
GPU model and memory
Exact command to reproduce

**Comment by karmel** [comment]
@derekjchow - do you have details on how the published numbers were generated? Some questions on how best to reproduce here.

**Comment by snownus** [comment]
Same issue. How to get the tensorflow minival 8k?

**Comment by ziyanyang** [comment]
> Same issue. How to get the tensorflow minival 8k?

Yes, I also have same issue.

**Comment by parvizp** [comment]
I was confused by the 8k number in the config files as well, but apparently its just an internal split they use: https://github.com/tensorflow/models/issues/5059#issuecomment-420532746

However, I'm still not able to reproduce the mAP scores reported in [the zoo](https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/detection_model_zoo.md#coco-trained-models) 
For instance, on the 2017 minival (5k images) for `ssd_mobilenet_v1_coco` I get:
```
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.263
 Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=100 ] = 0.421
 Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=100 ] = 0.278
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.017
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.131
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.510
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=  1 ] = 0.240
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 10 ] = 0.343
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.362
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.043
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.253
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.644
```
which is much better than the reported mAP of 21?

@pkulzc can you comment on the reported mAP numbers were produced?

**Comment by pkulzc** [comment]
This is because some of your coco 2017 eval images might appear in our training dataset.

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
